UnityProject/Assets/Scripts/Items/Medical/DefibrillatorPaddles.cs

```diff
@@ -17,7 +17,8 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 	{
 		if (DefaultWillInteract.Default(interaction,side) == false)
 			return false;
-		if (interaction.TargetObject.GetComponent<LivingHealthMasterBase>() == null)
+		var livingHealthMaster = interaction.TargetObject.GetComponent<LivingHealthMasterBase>();
+		if (livingHealthMaster == null)
 			return false;
 		var equipment = interaction.Performer.GetComponent<Equipment>();
 		var ObjectInSlot = equipment.GetClothingItem(NamedSlot.back).GameObjectReference;
```

Comment:
Possible runtime if a mob is the target? Or do they have Equipment component too?

Reply:
the equipment check is for the player using the defibrillator, not the target. It's to check if they have it equipped and aren't running around with only the paddles

Reply:
D'oh. Of course.